tests/Controller/JsonApiTestCase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Sylius\Tests\Controller;
+
+class JsonApiTestCase extends \Lakion\ApiTestCase\JsonApiTestCase
```

Comment:
I think an interface with just constants is less flexible (and quite an anti-pattern)
What about a helper class?
````
namespace Sylius\Tests\Controller;

class HttpHelper
{
    const HTTP_AUTHORIZATION = 'Bearer SampleToken...';

    const AUTHORIZATION_HEADER_WITH_CONTENT_TYPE = [
        'HTTP_Authorization' => self::HTTP_AUTHORIZATION,
        'CONTENT_TYPE' => 'application/json',
    ];

    const AUTHORIZATION_HEADER_WITH_ACCEPT = [
        'HTTP_Authorization' => self::HTTP_AUTHORIZATION,
        'ACCEPT' => 'application/json',
    ];
}
````
This way it can be refactored later (if needed) to `JsonHttpHelper` and `XmlHttpHelper` or `OAuthHttpHeader` and `PasswordHttpHeader` and so on.. And take advantage of constants override possibility which is not available with interfaces.